app/models/tutor.rb

```diff
@@ -1,8 +1,6 @@
 class Tutor < ApplicationRecord
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
 
-  before_save { email.downcase! }
```

Comment:
Since the uniqueness check isn't case sensitive it seems unnecessary to force downcase on email addresses. Some people like using upper case in their email addresses. I don't have strong feelings either way.